README.md

```diff
@@ -48,6 +51,34 @@ pacman -S base-devel git ncurses lib32-zlib gawk time unzip perl-xml-libxml \
  flex wget gettext quilt python2 openssl
 ```
 
+
+### Build-System-Structure  
+
+Where can I change something? What does these files do? 
+
+```
+:: Settings
+
+config.mk             - select what package lists to build
+modules               - define git repository, revision, etc of OpenWRT and Packages
+
+patches/              - Patches against the OpenWRT directory 
+packagelists/         - Package Lists 
+profiles/             - List of router profiles for each architecture - names come from the Imagebuilder
+
```

Comment:
probably you like to mention the "embedded-files" also, see: https://github.com/freifunk-berlin/firmware/commit/a66896d3ca02d1aff22be0c5b77e812ebd7596d4

Reply:
Nice feature. Do we use it in current builds? I did not see the directory `$(FW_DIR)/embedded-files` in the current master branch. Is it constructed at build-time?

Reply:
* https://git.wiki.kernel.org/index.php/GitFaq#Can_I_add_empty_directories.3F
I personally use it to include my ssh-public key in my test-builds (https://github.com/SvenRoederer/freifunk-berlin-firmware/tree/master/embedded-files/etc/dropbear), but I would not use is that much. 
There was an approach to use it for the banner, but this did not work as expected (https://github.com/freifunk-berlin/firmware/commit/c2573c99731aa9649327160c617e53d31e22da3d). Currently PR https://github.com/freifunk-berlin/firmware/pull/792 makes some use of it.

Reply:
Done.